### DISCUSSION TITLE
Search Results filtering

### DISCUSSION THREAD

**Discussion by GregBirbrayer** [microsoft-search/pnp-modern-search#4144] [Q&A]
HI, 
Microsoft search results allow filtering by the content type "Site". 
Would it be possible to introduce the same capability in pnpSearch?
For what we need - it is a "show stopper" :(
Thank you.

**Comment by kasperbolarsen** [comment]
You should be able to map the crawled properties behind the Managed property SiteTitle or SiteName ( can't remember the exact name) to a RefinableString

**Comment by kasperbolarsen** [comment]
Like this ( it hasn't reindexed all yet) 

![image](https://github.com/user-attachments/assets/f1c0fb62-864d-4273-a3fb-69c0d46215b6)



![image](https://github.com/user-attachments/assets/7be1278b-5724-4bfd-9390-5e6f5f91ba06)
